eng/pipelines/mono/templates/workloads-build.yml

```diff
@@ -42,6 +42,12 @@ jobs:
         value: '/p:OfficialBuildId=$(Build.BuildNumber)'
     - name: SignType
       value: $[ coalesce(variables.OfficialSignType, 'real') ]
+    - name: monoRepoRoot
```

Comment:
I double checked a binlog for passing `Build Workloads` step and did not find any mention of this property in anything that was part of `Find in files`.  I did find a single hit for each of the others you've included.

Also checked a runtime search and didn't find it. https://github.com/search?q=repo%3Adotnet%2Fruntime%20monoRepoRoot&type=code

Reply:
(none of the other variables in https://github.com/dotnet/runtime/pull/99179/files#diff-ef17aa4ba5069df7ab344cb6bdd121032a667b899762be670997df540a4313bd had any hits for me)